cura/CuraApplication.py

```diff
@@ -429,9 +438,11 @@ def __addAllEmptyContainers(self) -> None:
         self._container_registry.addContainer(cura.Settings.cura_empty_instance_containers.empty_quality_changes_container)
         self.empty_quality_changes_container = cura.Settings.cura_empty_instance_containers.empty_quality_changes_container
 
-    # Initializes the version upgrade manager with by providing the paths for each resource type and the latest
-    # versions.
     def __setLatestResouceVersionsForVersionUpgrade(self):
+        """Initializes the version upgrade manager with by providing the paths for each resource type and the latest
```

Comment:
Too long title line, but since the original was too long and this is supposed to be an automated conversion, it's ok

Reply:
Well I said in my commit that I manually changed and checked them, so I should have caught this one as well. I will change it though. Purely to satisfy my own OCD ;-)